src/content/en/2020/javascript.md

```diff
@@ -21,3 +21,657 @@ queries: 02_JavaScript
+        <td>Moment.js, jQuery</td>
+        <td class="numeric">32,793</td>
+        <td class="numeric">0.5%</td>
+      </tr>
+      <tr>
+        <td>FancyBox, jQuery, jQuery Migrate</td>
+        <td class="numeric">31,259</td>
+        <td class="numeric">0.5%</td>
+      </tr>
+      <tr>
+        <td>MooTools, jQuery, jQuery Migrate</td>
+        <td class="numeric">28,795</td>
+        <td class="numeric">0.5%</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <figcaption>
+    {{ figure_link(
+      caption="The most popular combinations of libraries and frameworks on mobile pages.",
+      sheets_gid="795160444",
+      sql_file="frameworks_libraries_combos.sql"
+    ) }}
+  </figcaption>
+</figure>
+
+We do also see a fair amount of more "modern" frameworks, like React, Vue and Angular—paired with jQuery, like as a result of either migration or third-parties including one or the other.
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Combination</th>
+        <th>Without jQuery</th>
+        <th>With jQuery</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>GSAP, Lodash, React, RequireJS, Zepto</td>
+        <td class="numeric">1.0%</td>
+        <td>&nbsp;</td>
+      </tr>
+      <tr>
+        <td>React, jQuery</td>
+        <td>&nbsp;</td>
+        <td class="numeric">0.6%</td>
+      </tr>
+      <tr>
+        <td>React</td>
+        <td class="numeric">0.4%</td>
+        <td>&nbsp;</td>
+      </tr>
+      <tr>
+        <td>React, jQuery, jQuery Migrate</td>
+        <td>&nbsp;</td>
+        <td class="numeric">0.4%</td>
+      </tr>
+      <tr>
+        <td>Vue.js, jQuery</td>
+        <td>&nbsp;</td>
+        <td class="numeric">0.3%</td>
+      </tr>
+      <tr>
+        <td>Vue.js</td>
+        <td class="numeric">0.2%</td>
+        <td>&nbsp;</td>
+      </tr>
+      <tr>
+        <td>AngularJS, jQuery</td>
+        <td>&nbsp;</td>
+        <td class="numeric">0.2%</td>
+      </tr>
+      <tr>
+        <td>GSAP, Hammer.js, Lodash, React, RequireJS, Zepto</td>
+        <td class="numeric">0.2%</td>
+        <td>&nbsp;</td>
+      </tr>
+    </tbody>
+    <tfoot>
+      <tr>
+        <th>Grand Total</th>
+        <th class="numeric">1.7%</th>
+        <th class="numeric">1.4%</th>
+      </tr>
+    </tfoot>
+  </table>
+  <figcaption>
+    {{ figure_link(
+      caption="The most popular combinations of React, Angular, and Vue with and without jQuery.",
+      sheets_gid="795160444",
+      sql_file="frameworks_libraries_combos.sql"
+    ) }}
+  </figcaption>
+</figure>
+
+More importantly, all these tools typically mean more code and more processing time.
+
+Looking specfically at the frameworks in use, we see that the median JavaScript bytes for pages using them varies dramatically depending on _what_ is being used.
+
+The graph below shows the median bytes for pages where any of the top 35 most commonly detected frameworks were found, broken down by client.
+
+{# Median bytes by JS framework #}
```

Comment:
@tkadlec do we have this data already?

Reply:
Yup! https://docs.google.com/spreadsheets/d/1cgXJrFH02SHPKDGD0AelaXAdB3UI7PIb5dlS0dxVtfY/edit#gid=1206934500

Reply:
Oh perfect, I didn't see the new queries in this PR 👍